spec/models/racer_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'rails_helper'
+
+RSpec.describe Racer, type: :model do
+  describe 'Validations' do
+    it { is_expected.to validate_presence_of(:name)    }
+    it { is_expected.to validate_presence_of(:born_at) }
+
+    context 'when trying to register an underaged racer' do
```

Comment:
You also need to make a context for the valid case (when pilot isn't underaged).